app/workers/form5655/vba_submission_job.rb

```
@@ -5,13 +5,26 @@
 module Form5655
   class VBASubmissionJob
     include Sidekiq::Worker
+    include SentryLogging
 
     sidekiq_options retry: false
 
+    sidekiq_retries_exhausted do |job, _ex|
+      user_uuid = job['args'][1]
+      UserProfileAttributes.find(user_uuid).destroy
+    rescue => e
+      log_message_to_sentry(
+        "Failed to delete user attributes #{user_uuid}",
+        :error,
+        exception: e.message
+      )
+    end
+
     def perform(submission_id, user_uuid)
       submission = Form5655Submission.find(submission_id)
-      user = User.find(user_uuid)
+      user = UserProfileAttributes.find(user_uuid)
```

you might want to fail fast if `user` doesn't exist here